lib/src/base-styles/_nx-colors.scss

```diff
@@ -184,19 +198,19 @@
 
     --nx-color-interactive-border: var(--nx-swatch-indigo-70);
     --nx-color-interactive-border-hover: var(--nx-swatch-indigo-60);
-    --nx-color-interactive-border-disabled: var(--nx-swatch-indigo-40);
+    --nx-color-interactive-border-disabled: var(--nx-color-disabled);
```

Comment:
Properties that are set the same in light and dark mode don't need to be repeated in the dark mode block